doc/nvim-tree-lua.txt

```diff
@@ -795,7 +795,10 @@ Appends a trailing slash to folder names.
 
 *nvim-tree.renderer.group_empty*
 Compact folders that only contain a single folder into one node.
-  Type: `boolean`, Default: `false`
+Can be a custom function that takes one argument (the relative path
```

Comment:
```suggestion
Boolean or a function that takes single argument (the relative path
```

Reply:
I agree with all the corrections, I would just leave "one" instead of "single" since it's more consistent with two other occurrences of something like "the function takes _one_ argument" throughout the docs

Reply:
Absolutely, consistency is the key. Thanks!